data/cookbook/make-http-post-request-bearer-auth/00-cohttp_lwt.ml

```diff
@@ -0,0 +1,72 @@
+---
+packages:
+  - name: "lwt"
+    tested_version: "5.7.0"
+    used_libraries:
+      - lwt
+  - name: "cohttp"
+    tested_version: "5.3.1"
+    used_libraries:
+      - cohttp
+  - name: "cohttp-lwt-unix"
+    tested_version: "5.3.1"
+    used_libraries:
+      - cohttp-lwt-unix
+  - name: "tls-lwt"
+    tested_version: "0.17.5"
+    used_libraries:
+      - tls-lwt
+discussion: |
+  - **REST API Operations - POST:** A `POST` operation creates a *new* resouce. Example: Add data to a database
+  - **SSL-TLS Exception:** Running this example may result in an error: `Exception: Failure No SSL or TLS support compiled into Conduit`. To resolve this issue you can run `opam install tls-lwt`
+  - **Reference:** The code below uses the GitHub REST API as an example. Please review the documentation here: [github.com/restap/issues/comments](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#create-an-issue-comment)
+---
+
+open Lwt
+open Cohttp
+open Cohttp_lwt_unix
+
+(* create the message payload for our POST request. This can vary from API to API so be
```

Comment:
```suggestion
(* Create POST request message payload. This can vary from API to API so be
```

> This can vary from API to API so be sure to review the API's documentation on what it expects to receive

This isn't very clear, feels a bit verbose too. What do really want to say here?